dataprofiler/tests/profilers/utils.py

```diff
@@ -188,7 +188,10 @@ def assert_profiles_equal(actual, expected):
     for actual_value, expected_value in zip(
         actual_dict.values(), expected_dict.values()
     ):
-        assert type(actual_value) == type(expected_value)
+        try:
+            assert type(actual_value) == type(expected_value)
+        except:
+            pass
```

Comment:
This should fail if they don't match.

Reply:
weird ... I'm almost positive Kenny had taken this out previously

Reply:
totally agree it should be out. I'll push a commit to resolve this at least

Reply:
resolved